### ISSUE TITLE
missing textures on beds, enderchests and shulkerboxes

### ISSUE THREAD

**Issue by stefanJ2BVB** [sp614x/optifine#5992]
## Description of Issue
When holding a bed, enderchest or shulkerbox, They look normal in first person perspective but look like missing textures in any other perspective. I've managed to find the cause of the problem is the emmissives.properties file in my pack, for some reason removing that file fixes the issue. 

## Steps to Reproduce
1. Log in to world with optifine and BetterVanillaBuilding <https://www.curseforge.com/minecraft/texture-packs/bettervanillabuilding>
2. hold any of the 3 mentioned items
3. look at yourself in F5

## OptiFine Version
Latest version (and all before)

## Installation Method
Standalone installer

## Prior Testing
- Does this happen in Vanilla without OptiFine?
no

![image](https://user-images.githubusercontent.com/80390033/124911949-a4334a00-dfed-11eb-9a4c-41afc78208bd.png)
![image](https://user-images.githubusercontent.com/80390033/124911977-ab5a5800-dfed-11eb-82b5-43865de347fa.png)
![image](https://user-images.githubusercontent.com/80390033/124912173-ebb9d600-dfed-11eb-8b35-ccf99ec5e2ce.png)

**Comment by Whitelisted1** [comment]
Does this only happen with the mod active? And does this happen with BetterVanillaBuidling without optifine

**Comment by stefanJ2BVB** [comment]
Yes, the issue only happens with the pack on. But turning off optifine fixes it

**Comment by Ari-RERA-43** [comment]
> Does this only happen with the mod active? And does this happen with BetterVanillaBuidling without optifine

They said the issue is with emissives so it's definately an OptiFine issue. Also, it's a resource pack, not a mod.

**Comment by Whitelisted1** [comment]
> > Does this only happen with the mod active? And does this happen with BetterVanillaBuidling without optifine
> 
> They said the issue is with emissives so it's definately an OptiFine issue. Also, it's a resource pack, not a mod.

Ah sorry, I'm not familiar

**Comment by JimStoneCraft** [comment]
This is the same issue I posted last year, still not fixed.

https://github.com/sp614x/optifine/issues/4964
